### PR TITLE
VM-Right-Sizing example 

### PR DESCRIPTION
**Pull request by dalino97**
Adding our vm right sizing example from [the community tips and tricks episode #13 ](https://www.youtube.com/watch?v=dGWlnd1lNGQ). This was done manually, however, I've automated it with Monaco and a colleague suggested that I upload this as an example here. 
The repo consists of 3 smaller projects - slack, jira and workflows. It also contains the delete file to be used with the delete command and the script that will take the values and import them as env variables. 
If there are any questions, please let me know.

### CODE REVIEW

File: VM-RIght-Sizing/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Dynatrace VM Right-Sizing Example
+
+This repository will help you deploy a configuration that will right-size your infrastructure using Dynatrace Automation Workflows. All configuration is deployed automatically using Dynatrace Configuration as Code solution Monaco.
+
+## Description 
+
+The example will help you detect improperly sized hosts, trigger an approval workflow in external tools such as Jira, ServiceNow ... and then react to that approval with a workflow to right-size the environment based on metadata stored in tags by calling the respective infrastructure automation APIs such as Azure, GCP, AWS, Kubernetes, OpenShift, VMWare, Ansible.
+
```
Comment (UnseenWizzard) [inline VM-RIght-Sizing/README.md:8]: I'd add a link to the community tips and tricks episode here as well. 

```suggestion

To learn more about this use case, check out the [Dynatrace Tips & Tricks Community Episode](https://youtu.be/dGWlnd1lNGQ).

```

File: VM-RIght-Sizing/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    path: second-example
+  - name: workflow-calculate-ticket
+    path: third-example
+
+environmentGroups:
+  - name: production
+    environments:
+      - name: production-environment
+        url:
+          value: "" # Enter environment ID e.g. https://<tenantId>.apps.dynatrace.com/
```
Comment (UnseenWizzard) [inline VM-RIght-Sizing/manifest.yaml:15]: URLs can be defined as environment variables as well. 
As you're asking users to fill out the shell scrip already I think it'd make it easier if they can just define their URL there as well :) 

```suggestion
          type: environment
          value: DT_URL
```

and then including the DT_URL variable and description how it should look in `input-needed.sh`

File: VM-RIght-Sizing/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            type: environment
+            name: DT_API_TOKEN
+          oAuth:
+            clientId:
+              type: environment
+              name: DT_OAUTH_CLIENT_ID
+            clientSecret:
+              type: environment
+              name: DT_OAUTH_CLIENT_SECRET
+            tokenEndpoint:
```
Comment (UnseenWizzard) [inline VM-RIght-Sizing/manifest.yaml:27]: The `tokenEndpoint` isn't needed for production environments (defaults to `https://sso.dynatrace.com/sso/oauth2/token`) and is generally just used against different stages of SSO internally. 

I would remove this from the public sample.

File: VM-RIght-Sizing/script/input-needed.sh [hunk 4]
```diff
@@ -0,0 +1,44 @@
+#!/bin/zsh 
+#  ^^ - change to SH if needed
```
Comment (UnseenWizzard) [inline VM-RIght-Sizing/script/input-needed.sh:2]:
```suggestion
```

File: VM-RIght-Sizing/README.md (hunk 1 above)
Comment (UnseenWizzard) [inline VM-RIght-Sizing/README.md:8]: Please add a few lines on what the example will actually do - how will it decide to right-size, what does it do with the slack/jira connections, etc.

File: VM-RIght-Sizing/manifest.yaml (hunk 3 above)
Comment (dalino97) [reply VM-RIght-Sizing/manifest.yaml]: Removed the token endpoint as well as the env variable in input-needed.sh.

File: VM-RIght-Sizing/README.md (hunk 1 above)
Comment (dalino97) [reply VM-RIght-Sizing/README.md:8]: Added a detailed description of what each workflow will do. Please let me know if that's okay.

Comment (UnseenWizzard) [reply VM-RIght-Sizing/README.md:8]: That is great, thanks!

**Review by UnseenWizzard — APPROVED** [review]

**Pull request merged**